libr/util/str.c

```diff
@@ -272,6 +272,25 @@ R_API char *r_str_home(const char *str) {
 	return dst;
 }
 
+R_API char *r_str_r2_prefix(const char *str) {
+	const char *prefix = r_sys_prefix (NULL);
+	size_t length;
+	length = strlen (prefix) + 1;
+	if (str) {
+		length += strlen (R_SYS_DIR) + strlen (str);
+	}
+	char *dst = (char *)malloc (length);
+	if (!dst) {
+		return NULL;
+	}
+	strcpy (dst, prefix);
```

Comment:
i would just use return r_str_newf("%s%s%s", r_sys_prefix(NULL), R_SYS_DIR, str);

Reply:
this function can be done in 1 line